CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Documentation for hipRAND is available at
 [https://rocm.docs.amd.com/projects/hipRAND/en/latest/](https://rocm.docs.amd.com/projects/hipRAND/en/latest/).
 
+## (unreleased) hipRAND x.y.z for ROCm 7.0
```

Comment:
It should be noted that I'm not sure what the correct hipRAND and ROCm version to put here is, so that will still need to be edited.

Reply:
Hi @umfranzw, thank you for the detailed instructions. I've applied your suggestions. Stanley also asked the team in an email to update the CMakeLists.txt file, but I don't see anything in CMakeLists.txt that needs to be changed. Please do let me know if you see something there that needs changing however. Thanks.

Reply:
My apologies, I forgot the packaging version was already updated for the now-defunct ROCm 6.5 release.  All is good.  Thanks!